### DISCUSSION TITLE
Welcome to DeepLearningTools Discussions!

### DISCUSSION THREAD

**Discussion by phborba** [phborba/DeepLearningTools#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by roya0045** [comment]
Looking good, I wanted to do something along those line. Though I think an important part that I wanted to look into is to go for a tiling approach with data. Tiling is critical as it helps split up data, ensure constant size, and can be extent instead of being restricted to a single image of full size. I'm pretty sure this is how ESRI did it on their end. I've looked into something like https://gdal.org/programs/gdal2tiles.html but the issue could be gathering the attributes and applying them back to the results to reimport the output tiles.
